src/js/components/DataTable/index.d.ts

```diff
@@ -113,6 +113,7 @@ export interface DataTableProps<TRowType = any> {
     show: number;
     count: number;
   }) => void;
+  verticalAlign?: 'middle' | 'top' | 'bottom';
```

Comment:
Should this be under `rowProps`?

Reply:
It appears that rowProps requires that you specify the primaryKey of the row you'd like to affect: https://v2.grommet.io/datatable#rowProps

I think we want this to apply across all. We could explore something like `defaultRowProps` that applies to all rows and has a key of vertical align. Wasn't sure if that gets confusing since the override is in `columns`

Reply:
doh... my memory can be so fuzzy :(

Reply:
As placed, I would expect it apply to header, body, and footer content. Although, I'm wondering if there might be designs that want the header aligned to "end" and the footer aligned to "start".

Reply:
Discussed and will go with:
```
verticalAlign: string | { header: , body: , footer: }
```

Reply:
Done.